pipeline/build-rpm-package.yaml

```diff
@@ -504,6 +510,10 @@ spec:
           value: $(tasks.rpmbuild-ppc64le.results.rpmbuild-artifact)
         - name: script-environment-image
           value: $(params.script-environment-image)
```

Comment:
Do you want to update script-environment-image now?

Reply:
Yeah, will need to to use --domain and --json-output

Reply:
The default environment-image is being set in here https://github.com/konflux-ci/rpmbuild-pipeline/blob/main/pipeline/build-rpm-package.yaml#L95